test/integration/ClientQueryTest.java

```diff
@@ -518,6 +521,23 @@ public void testSimpleExplanation() {
         }, READ, TypeDBOptions.core().infer(true).explain(true));
     }
 
+    @Test
+    public void testStreaming() {
```

Comment:
With these options (the default in TypeDB at time of writing), the server may respond with: 50 answers -> CONTINUE -> 1 answer [compensating for latency] -> DONE. The client will respond to CONTINUE with STREAM to keep iterating, and the server responds to STREAM with a 2nd DONE message.

This is expected and should be handled correctly (ie: ignored) by the client.